packages/components/src/components/action-group/action-group.tsx

```diff
@@ -157,6 +182,28 @@ export class ActionGroup extends LitElement {
 
   //#region Private Methods
 
+  private setActiveActionBySelectionMode(index: number, action: Action["el"]): void {
+    switch (this.selectionMode) {
+      case "multiple":
+        action.active = !action.active;
+        break;
+      case "single":
+        this.actionElements.forEach((a, i) => {
+          a.active = i === index ? !a.active : false;
+        });
+        break;
+      case "single-persist":
+        if (!this.actionElements[index].active) {
+          this.actionElements.forEach((a, i) => {
+            a.active = i === index;
+          });
+        }
+        break;
+      default:
```

Comment:
Suggestion: using `if` statements (with returns) might be more readable since it reduces nesting and avoids the empty default case handling.